src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCommercialButtons.tsx

```diff
@@ -124,9 +125,7 @@ export const ArtworkSidebarCommercialButtons: React.FC<React.PropsWithChildren<A
       context_owner_type: OwnerType.artwork,
       context_owner_slug: artwork.slug,
       context_owner_id: artwork.internalID,
-      signal_label: artwork.collectorSignals
-        ? getSignalLabel(artwork.collectorSignals)
-        : "",
+      signal_label: getSignalLabel(signals?.[artwork.internalID] ?? []),
```

Comment:
Would this populate the data when the `PrimaryLabelLine` component is not rendered with this `ArtworkSidebarCommercialButtons` component? I think we might still need a way to use the backend sourced signals. But I could be missing something here.